README.md

```diff
@@ -186,6 +187,42 @@ their place.
 
 **NOTE: Big-endian platforms are only supported as of release 0.4.0.**
 
+## Use in CMake build
+
+If your project uses the CMake build system, you can include cmake module
+`cmake/FlatccGenerateSources.cmake` that provides the following cmake
+function:
+
+    flatcc_generate_sources(GENERATED_SOURCE_DIRECTORY <output directory>
+                            GENERATE_BUILDER
+                            GENERATE_VERIFIER
+                            EXPECTED_OUTPUT_FILES <list of expected output files>
+                            DEFINITION_FILES <list of flatbuffer files (.fbs)>
+)
+
+`GENERATE_BUILDER` and `GENERATE_VERIFIER` are boolean options. When specified
+they will instruct flatcc to generate builder / verifier source code.
+
+Optionally you can let cmake know the directory where the flatcc executable
+is located in environment variable `FLATCC_BUILD_BIN_PATH`. This is especially
+usefull when cross-compiling. In that case you should provide the directory
```

Comment:
useful

Reply:
> Regarding minimum required version, there is reason why flatcc uses cmake_minimum_required (VERSION 2.8)
> 
> Otherwise it will not build on some older conservative platforms. If it is really necessary to use a more recent version and it only affects users own programs opting in to this feature then maybe OK, but otherwise it would interfere with portability which is important to FlatCC.

I had to update the cmake_minimum_required version from 2.8 to 3.1 in another recipe PR today because a conan-center hook rejects the PR if the cmake_minimum_required is not _at least_ 3.1.

In this case the minimum cmake version is 3.5 because from that version cmake function cmake_parse_arguments checks for value arguments that are miss their value, so it makes the function more fool-proof.

Reply:
> I'm suggesting to add an example of using this module within the FlatCC project, but perhaps not replacing existing since it could break portability with CMake version. I'm not sure if the module only works after CMake install and the README should probably also clarify how to make it available.

Ok, I will do that.
The only thing to do is that you have to do to make the module work is add the FlatCC cmake subdir to your CMAKE_MODULE_PATH.

Reply:
But if it works the same for correct arguments on earlier versions then an earlier MIN version  will still support the check for users with a recent CMake while not breaking the build for those who do not.